### PR TITLE
creating-new-cli-command-ec2rightsizing

### PR DESCRIPTION
**Pull request by CryptoPishka**
new Metricly CLI command: report ec2rightsizing

### CODE REVIEW

**Comment by rclagett** [comment]
LGTM!

File: ts/services/NewReportsService.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            {
+              "type": "diskio",
+              "rule": "none",
+              "fixedFloor": 0,
+              "requirePremiumIO": false
+            }
+          ]
+        }
+      });
+      //if (config.format === 'text' ) {
```
Comment (TheConnMan) [inline ts/services/NewReportsService.ts:62]: If it's commented out you might as well remove it.

File: ts/services/NewReportsService.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import * as moment from 'moment';
+import * as caporal from 'caporal';
+import * as request from 'request-promise';
+
+class NewReportsService {
+
+  public async ec2rightsizing(config, logger): Promise<void> {
+    logger.debug('\nListing the latest EC2 Right Sizing report');
```
Comment (TheConnMan) [inline ts/services/NewReportsService.ts:8]: Listing the report? "Listing" implies multiple reports, "report" is singular. How many reports are returned by this?

File: ts/services/NewReportsService.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
+import * as moment from 'moment';
+import * as caporal from 'caporal';
+import * as request from 'request-promise';
+
+class NewReportsService {
```
Comment (TheConnMan) [inline ts/services/NewReportsService.ts:5]: Is this because there's already a reports service?

**Review by TheConnMan — CHANGES_REQUESTED** [review]
Couple formatting errors from the build. Thanks for adding this!

File: ts/services/NewReportsService.ts (hunk 1 above)
Comment (CryptoPishka) [reply ts/services/NewReportsService.ts:62]: removing, ty

File: ts/services/NewReportsService.ts (hunk 2 above)
Comment (CryptoPishka) [reply ts/services/NewReportsService.ts:8]: changed to 'querying'

File: ts/services/NewReportsService.ts (hunk 3 above)
Comment (CryptoPishka) [reply ts/services/NewReportsService.ts:5]: yea, I decided to go this way. New reports use completely new APIs so why not we split them out? This way it looks better structured to me

**Review by TheConnMan — APPROVED** [review]

**Pull request merged**